app/views/auth/login.blade.php

```diff
@@ -6,7 +6,7 @@
             <div class="welcome-logo">
                 <img class="logo" height="50" src="{{ url('img/cachet-logo.svg') }}" alt="Cachet"/>
             </div>
-            {{ Form::open() }}
+            {{ Form::open(['url' => '/auth/login']) }}
```

Comment:
We should avoid coding urls like this because it breaks everything when people don't run cachet on the root domain.

We should always call the url helper method.

// cc @cachethq/owners